tests/Serializer/ContextTest.php

```diff
@@ -191,11 +193,5 @@ public function testSerializeNullOption()
 
         $context->setSerializeNull(true);
         $this->assertTrue($context->shouldSerializeNull());
-
-        $context->setSerializeNull("foo");
-        $this->assertTrue($context->shouldSerializeNull());
-
-        $context->setSerializeNull("0");
-        $this->assertFalse($context->shouldSerializeNull());
```

Comment:
This is BC break-ish, incompatible with strict typing by design.

Reply:
agree with getting rid of it